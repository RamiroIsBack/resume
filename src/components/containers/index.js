import IntroContainer from "./IntroContainer.js";

import SummaryContainer from "./SummaryContainer.js";

import ServicesContainer from "./ServicesContainer.js";

import TimeLineContainer from "./TimeLineContainer.js";

import SideBarContainer from "./SideBarContainer.js";

import NombreContainer from "./NombreContainer";

import FooterContainer from "./FooterContainer.js";

import MapaContainer from "./MapaContainer.js";

import JobsListContainer from "./JobsListContainer";

import ModalWorkContainer from "./ModalWorkContainer";

export {
  ModalWorkContainer,
  JobsListContainer,
  MapaContainer,
  FooterContainer,
  NombreContainer,
  SummaryContainer,
  IntroContainer,
  ServicesContainer,
  TimeLineContainer,
  SideBarContainer
};
