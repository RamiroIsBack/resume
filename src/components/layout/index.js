import App from "./App.js";
import JobsLayout from "./JobsLayout";
export { App, JobsLayout };
