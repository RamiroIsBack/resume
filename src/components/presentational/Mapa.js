import React, { Component } from 'react'
import { withScriptjs, withGoogleMap, GoogleMap, Marker,  InfoWindow, } from 'react-google-maps'

class Mapa extends Component {
  constructor(){
    super();
    this.state={
      mapa: null
    };
  }

  makeMarker(){
    let foto ='';
    let marker = {};
    if(this.props.mapInfo){
      foto = this.props.mapInfo.urlPic;
    }
    marker =(
      <Marker
        position={{
          lat: 39.3311347
          ,
          lng: -76.6450283
        }}
        onClick={() => this.handleClick('https://www.google.com/maps/place/Hampden,+Baltimore,+Maryland+21211/@39.3399061,-76.6020314,12.93z/data=!4m5!3m4!1s0x89c804d5f7905ff3:0x5cc1e5ec25566796!8m2!3d39.3333706!4d-76.632607')}
      >

        <InfoWindow >
          <div style = {{maxWidth: 600,padding : 0}}>

            <img role='presentation' src={foto} className ='infoWindow__img' alt='foto' id='fotoInfoWindow'
              style = {{
                opacity: 0.9, maxHeight: 60, maxWidth:60, cursor: 'pointer',float:'left', marginRight:'2px',
              }}
              onClick={() => this.handleClick('https://www.google.com/maps/place/Hampden,+Baltimore,+Maryland+21211/@39.3399061,-76.6020314,12.93z/data=!4m5!3m4!1s0x89c804d5f7905ff3:0x5cc1e5ec25566796!8m2!3d39.3333706!4d-76.632607')}>
            </img>

            <div className='infoWindow__bio' style = {{padding : 0,float:'right'}} >
              <h6 style = {{marginTop : 2,marginBottom:2}}>I'm in</h6>
              <h5 style = {{marginTop : 2,marginBottom:0}}>Hampden</h5>
              <h6 style = {{marginTop : 2,marginBottom:2}}>Baltimore, MD</h6>
            </div>

          </div>
        </InfoWindow>

      </Marker>

    );




    return marker;
  }
  handleClick(urlToGo){
    window.open(urlToGo,'_blank');
  }


  render() {

    let marker = this.makeMarker();
    return (

      <GoogleMap

        defaultZoom={9}
        defaultCenter={{  lat: 39.4832391, lng: -76.660066  }}
      >
        {marker}
      </GoogleMap>
    )
  }
}

export default withGoogleMap(Mapa)
