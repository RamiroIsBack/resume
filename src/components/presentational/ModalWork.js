import React from "react";
import style from "./styles";

class ModalWork extends React.Component {
  constructor() {
    super();
    this.state = {
      recibir: true
    };
  }

  gestionaIrACreacion(event) {
    if (event.target.id) {
      this.props.onSelectCreaciones(event.target.id);
    } else {
      this.props.onSelectCreaciones("allCreaciones");
    }
  }
  gestionaIrAFeria(event) {
    this.props.onSelectFerias("allFerias");
  }
  render() {
    // Render nothing if the 'show' prop is false
    if (!this.props.show) {
      return null;
    }

    var tipoList = this.props.creacionList.map((tipo, i) => {
      let estilo = {
        cursor: "pointer",
        color: "black",
        backgroundColor: "transparent",
        textDecoration: "none",
        padding: 0
      };

      return (
        <div key={i} style={{ padding: 0 }}>
          <div className="col-xs-offset-3 col-xs-9" style={{ padding: 1 }}>
            <a
              id={tipo.nombre}
              onClick={this.gestionaIrACreacion.bind(this)}
              to="/Diseños"
              style={estilo}
            >
              <h5 id={tipo.nombre}>{tipo.nombre}</h5>
            </a>
          </div>
        </div>
      );
    });

    var stiloModal = {
      position: "absolute",
      backgroundColor: "white",
      maxWidth: "100%",
      minHeight: 300,
      maxHeight: "100%",
      margin: "0 auto",
      //To make sure that it is really on top of all other elements, we set z-index to a high value (by default all elements have a z-index value of 0).
      zIndex: 6668,
      top: "20px",
      left: "2px",
      right: "70px",
      bottom: "40px",
      overflow: "auto",
      WebkitOverflowScrolling: "touch",
      borderRadius: "5px",
      outline: "none"
    };

    return (
      <div style={style.modal.backdropStyle}>
        <div style={stiloModal}>
          <div>
            <div
              className="col-xs-12 col-sm-8 col-md-8 col-lg-8"
              style={style.modal.formContainer}
            >
              <button
                onClick={this.props.onClose}
                className="btn glyphicon glyphicon-remove pull-right"
                style={style.modal.btnClose}
              />
              <div className="col-xs-offset-1 col-xs-8">
                <a
                  id="allCreaciones"
                  name="allCreaciones"
                  onClick={this.gestionaIrACreacion.bind(this)}
                  to="/Diseños"
                  style={{
                    cursor: "pointer",
                    color: "black",
                    backgroundColor: "transparent",
                    textDecoration: "none",
                    padding: 0
                  }}
                >
                  <h4 id="allCreaciones" name="allCreaciones">
                    {" "}
                    TIENDA:
                  </h4>
                </a>
              </div>

              <div>{tipoList}</div>

              <div className="col-xs-offset-1 col-xs-8">
                <a
                  to="/Conocenos"
                  style={{
                    cursor: "pointer",
                    color: "black",
                    backgroundColor: "transparent",
                    textDecoration: "none",
                    padding: 0
                  }}
                  onClick={this.props.onClose}
                >
                  <h4> SOBRE MI</h4>
                </a>
              </div>

              <div className="col-xs-offset-1 col-xs-8">
                <a
                  to="/Taller"
                  style={{
                    cursor: "pointer",
                    color: "black",
                    backgroundColor: "transparent",
                    textDecoration: "none",
                    padding: 0
                  }}
                  onClick={this.props.onClose}
                >
                  <h4> TALLER</h4>
                </a>
              </div>

              <div className="col-xs-offset-1 col-xs-8">
                <a
                  id="creaciones"
                  to="/Ferias"
                  style={{
                    cursor: "pointer",
                    color: "black",
                    backgroundColor: "transparent",
                    textDecoration: "none",
                    padding: 0
                  }}
                  onClick={this.props.onClose}
                >
                  <h4> FERIAS</h4>
                </a>
              </div>

              <div className="col-xs-offset-1 col-xs-8">
                <a
                  to="/Contacto"
                  style={{
                    cursor: "pointer",
                    color: "black",
                    backgroundColor: "transparent",
                    textDecoration: "none",
                    padding: 0
                  }}
                  onClick={this.props.onClose}
                >
                  <h4> CONTACTO</h4>
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    );
  }
}

//I tihk I don't need this cos it's only to put children to it
/*Modal.propTypes = {
  onClose: PropTypes.func.isRequired,
  show: PropTypes.bool,
  children: PropTypes.node
}*/

export default ModalWork;
