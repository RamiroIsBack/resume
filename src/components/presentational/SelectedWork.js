import React, { Component } from "react";
import Works_css from "../../css"; // eslint-disable-line no-unused-vars

class SelectedWork extends Component {
  render() {
    return (
      <div>
        <h2>{this.props.copy.nombre}</h2>
        <p>{this.props.copy.bio}</p>
      </div>
    );
  }
}

export default SelectedWork;
