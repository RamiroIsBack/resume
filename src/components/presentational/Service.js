import React, {Component} from 'react';
import LazyLoad from 'react-lazy-load';

class Service extends Component {

  handleClick(){
    if(this.props.serviceInfo.urlLink){
      window.open(this.props.serviceInfo.urlLink,'_blank');
    }
  }

  render() {
    let logo= '';
    let cursor = {};
    let stylingBarWidth =  { width:'80%'};
    if(this.props.serviceInfo){
      logo = this.props.serviceInfo.urlPic;
      let percentage = parseInt(this.props.serviceInfo.percentage,10);
      if (this.props.sectionSelected){
        stylingBarWidth ={
          width: ''+percentage+'%',
          animationName: 'progress',
          animationDuration: '3s',
          animationTimingFunction: 'ease-in-out',
        };
      }else {
        stylingBarWidth ={
          width: ''+percentage+'%',

        };
      }
      if(this.props.serviceInfo.urlLink){
        cursor={
          cursor:'pointer',
        };
      }
    }

    return (
      <div >
        <div className = 'name__pic__container'>
          <h2  className = 'service__name' >{this.props.serviceInfo.nombre}</h2>
          <LazyLoad height={60} offsetVertical={100}>
            <img className = 'logo_pic'
              style= {cursor}
              alt=''
              src= {logo}
              onClick = {this.handleClick.bind(this)}
            ></img>
          </LazyLoad>
        </div>
        <div className = 'progress__bar__container'>
          <div className = 'progress__bar' style={stylingBarWidth}></div>
        </div>
        <h5 className = 'percentage__num'>{this.props.serviceInfo.percentage}%</h5>
        <div className = 'bio__container__services'>
          {this.props.serviceInfo.bio.split('\n').map((item, key) => {
            return <span key={key}>{item}<br/></span>;})}
        </div>
      </div>
    );
  }
}

export default Service;
