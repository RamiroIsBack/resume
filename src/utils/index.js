import Firebase from './Firebase.js';
import PopularFunctions from './PopularFunctions.js';
export {
  Firebase,
  PopularFunctions,
};