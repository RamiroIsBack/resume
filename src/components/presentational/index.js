import Service from "./Service.js";
import TimeBox from "./TimeBox.js";
import Mapa from "./Mapa.js";
import Work from "./Work.js";
import SelectedWork from "./SelectedWork.js";
import ModalWork from "./ModalWork.js";
export { Mapa, Service, TimeBox, Work, SelectedWork, ModalWork };
