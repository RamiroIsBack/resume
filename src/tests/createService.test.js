/**
 * @jest-environment node
 */

import React from 'react';
import renderer from 'react-test-renderer';
import {Service} from '../components/presentational'
import {Firebase} from '../utils';

it('getServicesComponents', () => {
  return Firebase.getCopyForTesting()
    .then(result => { //the return is a promise: result(copyList)
      let component = {};
      let servicesComponentList = [];
      let animeIt =true;
      let rawList = result;
      let serviceRawListObject = {};
      for(let i = 0; i<rawList.length; i++){
        if(rawList[i].nombre ==='services'){
          serviceRawListObject = rawList[i];
        }
      }
      for (var property in serviceRawListObject) {
        if (serviceRawListObject.hasOwnProperty(property)) {
          if (property!=='nombre' && property !=='id'){
            //console.log(JSON.stringify(property));
            component = renderer.create(
              <div key ={property}>
                <Service
                  sectionSelected= {animeIt}
                  serviceInfo ={serviceRawListObject[property]}
                />
              </div>
            ).toJSON();
          }
        }
        servicesComponentList.push( component   );
      }

      expect (servicesComponentList).toMatchSnapshot();

    });
});
// @magnusart got it right
// https://www.bountysource.com/issues/41190983-syntax-error-invalid-or-unexpected-token-with-png
