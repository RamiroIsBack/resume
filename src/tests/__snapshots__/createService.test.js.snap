// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getServicesComponents 1`] = `
Array [
  <div>
    <div>
        <div
            className="name__pic__container"
        >
            <h2
                className="service__name"
            >
                Bootstrap
            </h2>
            <img
                className="logo_pic"
                onClick={[Function]}
                src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fbootstrap.png?alt=media&token=37ff7c85-72f7-4220-b2eb-19a0a232d355"
                style={Object {}}
            />
        </div>
        <div
            className="progress__bar__container"
        >
            <div
                className="progress__bar"
                style={
                    Object {
                        "animationDuration": "3s",
                        "animationName": "progress",
                        "animationTimingFunction": "ease-in-out",
                        "width": "75%",
                      }
                }
            />
        </div>
        <h5
            className="percentage__num"
        >
            75
            %
        </h5>
        <div
            className="bio__container__services"
        >
            <span>
                I've been using it since I started learning JavaScript, it's a very easy and useful tool for styling responsive elements.
                <br />
            </span>
            <span>
                 I've been using Css-grid lately though, it's great!
                <br />
            </span>
        </div>
    </div>
</div>,
  <div>
    <div>
        <div
            className="name__pic__container"
        >
            <h2
                className="service__name"
            >
                Css
            </h2>
            <img
                className="logo_pic"
                onClick={[Function]}
                src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fcss3.png?alt=media&token=91f3a9b3-d360-43f4-9277-c94bfae701e3"
                style={Object {}}
            />
        </div>
        <div
            className="progress__bar__container"
        >
            <div
                className="progress__bar"
                style={
                    Object {
                        "animationDuration": "3s",
                        "animationName": "progress",
                        "animationTimingFunction": "ease-in-out",
                        "width": "70%",
                      }
                }
            />
        </div>
        <h5
            className="percentage__num"
        >
            70
            %
        </h5>
        <div
            className="bio__container__services"
        >
            <span>
                because of my Java and C++ background, I tended to do everything programmatically but I recently discover the world of possibilities that Css offers right out of the box and I love it!
                <br />
            </span>
        </div>
    </div>
</div>,
  <div>
    <div>
        <div
            className="name__pic__container"
        >
            <h2
                className="service__name"
            >
                Firebase
            </h2>
            <img
                className="logo_pic"
                onClick={[Function]}
                src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Ffirebase.png?alt=media&token=23944ce2-4d4b-4384-890d-d6a577c843bd"
                style={Object {}}
            />
        </div>
        <div
            className="progress__bar__container"
        >
            <div
                className="progress__bar"
                style={
                    Object {
                        "animationDuration": "3s",
                        "animationName": "progress",
                        "animationTimingFunction": "ease-in-out",
                        "width": "80%",
                      }
                }
            />
        </div>
        <h5
            className="percentage__num"
        >
            80
            %
        </h5>
        <div
            className="bio__container__services"
        >
            <span>
                I've been using Firebase for all my recent projects, using its real-time data-base, authentication built-in process, functions on server-side.... It's a very complete tool.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div>
    <div>
        <div
            className="name__pic__container"
        >
            <h2
                className="service__name"
            >
                Github
            </h2>
            <img
                className="logo_pic"
                onClick={[Function]}
                src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2FgitTransInnerWhite.png?alt=media&token=2db6ffc5-a500-4747-a4c9-739294765497"
                style={
                    Object {
                        "cursor": "pointer",
                      }
                }
            />
        </div>
        <div
            className="progress__bar__container"
        >
            <div
                className="progress__bar"
                style={
                    Object {
                        "animationDuration": "3s",
                        "animationName": "progress",
                        "animationTimingFunction": "ease-in-out",
                        "width": "60%",
                      }
                }
            />
        </div>
        <h5
            className="percentage__num"
        >
            60
            %
        </h5>
        <div
            className="bio__container__services"
        >
            <span>
                I use it to store my projects and version control them.
                <br />
            </span>
            <span>
                I've worked a bit with different branches.
                <br />
            </span>
            <span>
                Feel free to clone this repository or to dive in the code clicking on github logo :)
                <br />
            </span>
        </div>
    </div>
</div>,
  <div>
    <div>
        <div
            className="name__pic__container"
        >
            <h2
                className="service__name"
            >
                HTML
            </h2>
            <img
                className="logo_pic"
                onClick={[Function]}
                src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fhtml.png?alt=media&token=9d3ccc3a-da88-47b2-8964-eb042b5f138d"
                style={Object {}}
            />
        </div>
        <div
            className="progress__bar__container"
        >
            <div
                className="progress__bar"
                style={
                    Object {
                        "animationDuration": "3s",
                        "animationName": "progress",
                        "animationTimingFunction": "ease-in-out",
                        "width": "60%",
                      }
                }
            />
        </div>
        <h5
            className="percentage__num"
        >
            60
            %
        </h5>
        <div
            className="bio__container__services"
        >
            <span>
                I used pure HTML for just few web pages, it's specially useful for me when dealing with a configuration page for the owner of the page to configure the web-app or to fill up the data-base with copy... very fast and straight forward.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div>
    <div>
        <div
            className="name__pic__container"
        >
            <h2
                className="service__name"
            >
                Jest
            </h2>
            <img
                className="logo_pic"
                onClick={[Function]}
                src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fjest2.png?alt=media&token=43e189b7-a2b3-4602-a4c9-83cd1980eef4"
                style={Object {}}
            />
        </div>
        <div
            className="progress__bar__container"
        >
            <div
                className="progress__bar"
                style={
                    Object {
                        "animationDuration": "3s",
                        "animationName": "progress",
                        "animationTimingFunction": "ease-in-out",
                        "width": "50%",
                      }
                }
            />
        </div>
        <h5
            className="percentage__num"
        >
            50
            %
        </h5>
        <div
            className="bio__container__services"
        >
            <span>
                Very recently I started using Jest with snapshots and is fantastic.
                <br />
            </span>
            <span>
                Here in this project I test few functions and components that way, clone this repo and check it out!
                <br />
            </span>
        </div>
    </div>
</div>,
  <div>
    <div>
        <div
            className="name__pic__container"
        >
            <h2
                className="service__name"
            >
                NodeJs
            </h2>
            <img
                className="logo_pic"
                onClick={[Function]}
                src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2FnodeJs.jpg?alt=media&token=4650d90d-96cb-4a2a-bd6d-8a62fd7e76db"
                style={Object {}}
            />
        </div>
        <div
            className="progress__bar__container"
        >
            <div
                className="progress__bar"
                style={
                    Object {
                        "animationDuration": "3s",
                        "animationName": "progress",
                        "animationTimingFunction": "ease-in-out",
                        "width": "20%",
                      }
                }
            />
        </div>
        <h5
            className="percentage__num"
        >
            20
            %
        </h5>
        <div
            className="bio__container__services"
        >
            <span>
                haven't use it that much jet, I'm certainly looking forward to it. I've used it for certain functions on the Firebase back-end, using modules like node-mailer to send emails to users and the like.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div>
    <div>
        <div
            className="name__pic__container"
        >
            <h2
                className="service__name"
            >
                ReactJs
            </h2>
            <img
                className="logo_pic"
                onClick={[Function]}
                src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2FreactJs.png?alt=media&token=e26d4e05-5609-4f14-8f34-2511419cddf8"
                style={Object {}}
            />
        </div>
        <div
            className="progress__bar__container"
        >
            <div
                className="progress__bar"
                style={
                    Object {
                        "animationDuration": "3s",
                        "animationName": "progress",
                        "animationTimingFunction": "ease-in-out",
                        "width": "80%",
                      }
                }
            />
        </div>
        <h5
            className="percentage__num"
        >
            80
            %
        </h5>
        <div
            className="bio__container__services"
        >
            <span>
                All the heavy lifting of my web-apps have been done so far with ReactJs, using react-Redux it's so powerful, I just love the component way of programming.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div>
    <div>
        <div
            className="name__pic__container"
        >
            <h2
                className="service__name"
            >
                ReactNative
            </h2>
            <img
                className="logo_pic"
                onClick={[Function]}
                src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Freact-Native.png?alt=media&token=478958c2-f605-491b-b08f-36920c139501"
                style={Object {}}
            />
        </div>
        <div
            className="progress__bar__container"
        >
            <div
                className="progress__bar"
                style={
                    Object {
                        "animationDuration": "3s",
                        "animationName": "progress",
                        "animationTimingFunction": "ease-in-out",
                        "width": "15%",
                      }
                }
            />
        </div>
        <h5
            className="percentage__num"
        >
            15
            %
        </h5>
        <div
            className="bio__container__services"
        >
            <span>
                I've just seen a glimpse of what can be done in react-native but I understand is the next step for me, I've got experience on mobile-app-development in Java for Android devices, that combined with my ReactJs experience points me towards a very enjoyable future time programming in React-Native :)
                <br />
            </span>
        </div>
    </div>
</div>,
  <div>
    <div>
        <div
            className="name__pic__container"
        >
            <h2
                className="service__name"
            >
                ReactNative
            </h2>
            <img
                className="logo_pic"
                onClick={[Function]}
                src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Freact-Native.png?alt=media&token=478958c2-f605-491b-b08f-36920c139501"
                style={Object {}}
            />
        </div>
        <div
            className="progress__bar__container"
        >
            <div
                className="progress__bar"
                style={
                    Object {
                        "animationDuration": "3s",
                        "animationName": "progress",
                        "animationTimingFunction": "ease-in-out",
                        "width": "15%",
                      }
                }
            />
        </div>
        <h5
            className="percentage__num"
        >
            15
            %
        </h5>
        <div
            className="bio__container__services"
        >
            <span>
                I've just seen a glimpse of what can be done in react-native but I understand is the next step for me, I've got experience on mobile-app-development in Java for Android devices, that combined with my ReactJs experience points me towards a very enjoyable future time programming in React-Native :)
                <br />
            </span>
        </div>
    </div>
</div>,
  <div>
    <div>
        <div
            className="name__pic__container"
        >
            <h2
                className="service__name"
            >
                ReactNative
            </h2>
            <img
                className="logo_pic"
                onClick={[Function]}
                src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Freact-Native.png?alt=media&token=478958c2-f605-491b-b08f-36920c139501"
                style={Object {}}
            />
        </div>
        <div
            className="progress__bar__container"
        >
            <div
                className="progress__bar"
                style={
                    Object {
                        "animationDuration": "3s",
                        "animationName": "progress",
                        "animationTimingFunction": "ease-in-out",
                        "width": "15%",
                      }
                }
            />
        </div>
        <h5
            className="percentage__num"
        >
            15
            %
        </h5>
        <div
            className="bio__container__services"
        >
            <span>
                I've just seen a glimpse of what can be done in react-native but I understand is the next step for me, I've got experience on mobile-app-development in Java for Android devices, that combined with my ReactJs experience points me towards a very enjoyable future time programming in React-Native :)
                <br />
            </span>
        </div>
    </div>
</div>,
]
`;
